packages/web-components/src/tab/tab.styles.ts

```diff
@@ -0,0 +1,117 @@
+import { css } from '@microsoft/fast-element';
+import { display } from '@microsoft/fast-foundation';
+import {
+  borderRadiusCircular,
+  borderRadiusMedium,
+  borderRadiusSmall,
+  colorCompoundBrandStroke,
+  colorNeutralForeground1,
+  colorNeutralForeground2,
+  colorNeutralForegroundDisabled,
+  colorNeutralStroke1Hover,
+  colorStrokeFocus1,
+  colorStrokeFocus2,
+  fontFamilyBase,
+  fontSizeBase300,
+  fontWeightSemibold,
+  lineHeightBase300,
+  spacingHorizontalM,
+  spacingHorizontalMNudge,
+} from '../theme/design-tokens.js';
+
+export const styles = css`
+  ${display('none')}
+
+  :host {
+    position: relative;
+    display: inline-flex;
+    flex-direction: column;
+    cursor: pointer;
+    box-sizing: border-box;
+    justify-content: center;
+    line-height: ${lineHeightBase300};
+    font-family: ${fontFamilyBase};
+    font-size: ${fontSizeBase300};
+    color: ${colorNeutralForeground2};
+    fill: currentcolor;
+    grid-row: 1;
+    padding: ${spacingHorizontalM} ${spacingHorizontalMNudge};
+    border-radius: ${borderRadiusMedium};
+  }
+  :host .tab-content {
+    display: inline-flex;
+    flex-direction: column;
+    padding: 0 2px;
+  }
+
+  :host([aria-selected='true']) {
+    color: ${colorNeutralForeground1};
+  }
+
+  :host([aria-selected='true']),
+  :host([aria-selected='true'][disabled]) {
```

Comment:
You probably dont need to specifically target disabled here unless its trying to set it own font-weight